alchemy-web/docs/concepts/state.md

```diff
@@ -92,9 +92,52 @@ const app = await alchemy("my-app", {
 });
 ```
 
+### Durable Objects State Store (Recommended)
+
+For high-performance cloud state storage, use DOStateStore with Cloudflare Durable Objects.
+
+```typescript
+import { DOStateStore } from "alchemy/cloudflare";
+
+const app = await alchemy("my-app", {
+  stage: "prod", 
+  phase: process.argv.includes("--destroy") ? "destroy" : "up",
+  stateStore: (scope) => new DOStateStore(scope, {
+    // Cloudflare API credentials
+    apiKey: alchemy.secret(process.env.CLOUDFLARE_API_KEY),
+    email: process.env.CLOUDFLARE_EMAIL,
```

Comment:
These are optional too and will be inferred from env variables or oauth. Add a callout pointing readers to the Cloudflare auth guide